SharkCage/CageManager/CageManager.cpp

```diff
@@ -223,7 +223,9 @@ bool CreateACL(std::unique_ptr<PSID, decltype(local_free_deleter<PSID>)> group_s
 	}
 
 	// We need in order to create the process.
-	STARTUPINFO info = { 0 };
+	STARTUPINFO info = { sizeof(info) };
```

Comment:
`STARTUPINFO info = {};` should be sufficient

Reply:
Thanks for the tip, I will try that.